video/vaapi.h

```diff
@@ -20,11 +20,11 @@
 
 #include <stdbool.h>
 #include <inttypes.h>
-#include <pthread.h>
```

Comment:
This doesn't seem to need pthreads either.

Reply:
See https://github.com/mpv-player/mpv/pull/12709#issuecomment-1793502512 for explenation why some stray `<pthread.h>` has to be there until new win32 impl.

Reply:
removed